src/clarity-angular/datagrid/render/column-resizer.ts

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Directive, ElementRef, Renderer, OnDestroy, AfterViewInit, Output, EventEmitter} from "@angular/core";
+import {DomAdapter} from "./dom-adapter";
+import {DatagridRenderOrganizer} from "./render-organizer";
+
+@Directive({
+    selector: "clr-dg-column"
+})
+export class DatagridColumnResizer implements AfterViewInit, OnDestroy {
+
+
+    constructor(private el: ElementRef,
+                private renderer: Renderer,
+                private organizer: DatagridRenderOrganizer,
+                private domAdapter: DomAdapter) {
+
+        this.columnEl = el.nativeElement;
+    }
+
+    columnEl: Element;
```

Comment:
`Element` seems weird here. I could see either `HTMLElement` or just `Node`. But either way, you can't actually type the native element because it would be different for each platform (browser, server-side, native mobile apps, ...). So I think `any` is safer as long as we use this in an Angular directive.